src/deps.txt

```diff
@@ -56,3 +56,6 @@ ignition
 
 # for grub install when creating images without anaconda
 grub2
+
+# For creating bootable UEFI media on x86_64
+shim-x64 grub2-efi-x64
```

Comment:
Is there a way to filter this according to the architecture? Something like
```
#X86_64 shim-x64 grub2-efi-x64
#AARCH64 shim-aa64 grub2-efi-aa64
```
Either that or use `shim grub2-efi` and `dnf` will choose the appropriate architecture.

Reply:
yeah - so with your work I anticipate we'll add filtering or separate files for arch specific dependencies. So I anticipate this work will get shuffled when you submit a fix for https://github.com/coreos/coreos-assembler/issues/350 and https://github.com/coreos/fedora-coreos-config/issues/53